contracts/scripts/clean.sh

```diff
@@ -0,0 +1,68 @@
+#!/bin/bash
+
+set -euo pipefail
+
+# --- FUNCTIONS
+
+function terminate_contract {
+  local contract_name=$1
+  local contract_dir=$2
```

Comment:
Quotes around variables that contains filesystem names please :)

Reply:
done